install/overlays/README.md

```diff
@@ -0,0 +1,98 @@
+
+# Kustomize Camel K
+
+Kustomize provides a declarative approach to the configuration customization of a Camel-K installation. Kustomize works either with a standalone executable or as a built-in to kubectl.
+
+Basic overlays are provided for easy usage.
+
+## HOW-TO
+
+### Initialize
+
+First create a new kustomization from the wanted version (kubernetes or openshift) in the repository:
+```sh
+kustomize create --resources https://github.com/apache/camel-k.git/install/overlays/kubernetes\?ref\=exp/kustomize_structure
```

Comment:
IMO, this one should be a separate chapter for user customization. We should not require anything but `kubectl` for a immediate user experiente. We miss a quickstart where the user run something like: `  # Build from github
  kubectl kustomize https://github.com/kubernetes-sigs/kustomize.git/examples/helloWorld?ref=v1.0.6 | kubectl create -f` where, if it's the case the use can choose between Kubernetes and Openshift, withouth knowing anything about overlays. What I mean is that we provide 2 procedures:
 * Kubernetes installation: `kubectl kustomize https://github.com/apache/camel-k.git/install/overlays/kubernetes | kubectl create -f` 
 *  Openshift installation: `kubectl kustomize https://github.com/apache/camel-k.git/install/overlays/openshift | kubectl create -f`